product_docs/docs/pem/9/certificates/replacing_ssl_certificates.mdx

```diff
@@ -2,137 +2,125 @@
+   ```
   
-9.  Change the permissions on the `root.crt` file:
+1. Change the owner and permissions on the `root.crt` file:
 
-    ```shell
-    chmod 600 /var/lib/edb/as<x>/data/root.crt
-    ```
+   ```shell
+   chown enterprisedb /var/lib/edb/as<x>/data/root.crt
+   chmod 600 /var/lib/edb/as<x>/data/root.crt
+   ```
 
-10. Use the `openssl_rsa_generate_crl()` function to create the certificate revocation list (`root.crl`):
+9. Use the `openssl_rsa_generate_crl()` function to create the certificate revocation list (`root.crl`):
 
-    ```shell
-    /usr/edb/as<x>/bin/psql -U enterprisedb -d pem --no-psqlrc -t -A -c
-     "SELECT openssl_rsa_generate_crl('/var/lib/edb/as<x>/data/ca_certificate.crt', '/var/lib/edb/as<x>/data/ca_key.key')" > /var/lib/edb/as<x>/data/root.crl
-    ```
+   ```shell
+   /usr/edb/as<x>/bin/psql -U enterprisedb -d pem --no-psqlrc -t -A -c
+   "SELECT openssl_rsa_generate_crl('/var/lib/edb/as<x>/data/ca_certificate.crt', '/var/lib/edb/as<x>/data/ca_key.key')" > /var/lib/edb/as<x>/data/root.crl
+   ```
 
-11. Change the permissions of the `root.crl` file:
+10. Change the permissions of the `root.crl` file:
 
     ```shell
     chmod 600 /var/lib/edb/as<x>/data/root.crl
     ```
 
-12. Use the `openssl_rsa_generate_key()` function to generate the `server.key` file:
+11. Use the openssl command to generate the `server.key` file:
 
     ```shell
-    /usr/edb/as<x>/bin/psql -U enterprisedb -d pem --no-psqlrc -t -A -c "SELECT public.openssl_rsa_generate_key(1024)" >> /var/lib/edb/as<x>/data/server.key
+    openssl genrsa -out server.key 4096 
     ```
 
-13.  `cat` the contents to the variable `SSL_KEY` for use when generating the `server.crt` file and change the permissions on the `server.key` file:
+12. Move the `server.key` to the data directory of the backend server and change the permissions:
 
     ```shell
-    SSL_KEY=$(cat /var/lib/edb/as<x>/data/server.key)
-
+    mv server.key /var/lib/edb/as<x>/data
     chmod 600 /var/lib/edb/as<x>/data/server.key
     ```
 
-14. Use the `SSL_KEY` to generate the server certificate. Save the certificate in the `server.crt` file. For simplicity, first place the SQL query into a temporary file with a unique name:
+13. Use the openssl req command to create the CSR:
 
-    ```shell
-    echo "SELECT openssl_csr_to_crt(openssl_rsa_key_to_csr('${SSL_KEY}',
-    'PEM','US', 'MA', 'Bedford', 'Postgres Enterprise Manager',
-    'support@enterprisedb.com'),
-    '/var/lib/edb/as<x>/data/ca_certificate.crt',
-    '/var/lib/edb/as<x>/data/ca_key.key')" > /tmp/_random.$$
+   ```shell
+   openssl req -new -key server.key -out server.csr -subj '/C=IN/ST=MH/L=Pune/O=EDB/CN=PEM'
+   ```
 
-    /usr/edb/as<x>/bin/psql -U enterprisedb -d pem --no-psqlrc -t -A -f /tmp/_random.$$ >> /var/lib/edb/as<x>/data/server.crt
+   Where `-subj` can be provided as per your requirements. You can define `CN` as hostname/domain name of the PEM server host. 
```

Comment:
Is the use of "can" indicating the `-subj` phrase is optional? If not, suggested change is:
```
Where `-subj` is provided as per your requirements. Define `CN` as hostname/domain name of the PEM server host. 
```